carwings.net/Models/VehicleLocator.cs

```diff
@@ -31,5 +31,11 @@ public class Location
 
         [JsonProperty("longitudeDMS")]
         public double Longitude { get; set; }
+
+        /// <summary>
+        /// Position can be AVAILABLE, and potentially other values.
+        /// </summary>
+        public String Position { get; set; }
```

Comment:
what does position represent?

Reply:
Good question - I don't know!  I've seen it be AVAILABLE and UNAVAILABLE.  I have no clue what it means.  We should probably figure it out.  Best way to do that is to add it to the data structure and log it on a car for days, or potentially across multiple Leafs for days.